aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -344,10 +344,23 @@ Tensor real(const Tensor& self) {
   }
 }
 
+Tensor neg_view(const Tensor& self) {
```

Comment:
@ezyang should we make this API public to let users get negative view if they want (even when not in complex world)?

Reply:
Maybe with a leading underscore. I wouldn't really advertise it all that much as there probably isn't very much benefit at all you can get from it.